public/javascripts/gformFileUpload.js

```diff
@@ -298,13 +308,16 @@
     }
 
     // Handle file upload request
-    function uploadFile(file, fileId) {
+    function uploadFile(file, fileId, formComponentId) {
       var formData = new FormData();
       formData.append(
         fileId,
         file,
         fileId + "_" + file.name.replace(/\\/g, "/").replace(/.*\//, "")
       );
+      $("#" + formComponentId + "-files")
+        .empty()
+        .append(startProgressBar(""));
```

Comment:
startProgressBar is a no-arg function, why passing ""?

Reply:
Fixed